ddp-workspace/projects/ddp-singular/src/styles/_common.scss

```diff
@@ -26,6 +26,7 @@ body {
   line-height: 1.5rem;
   color: $color-text;
   height: 100%;
+  background-color: $__cultured;
```

Comment:
The `background-color` should be applied for Dashboard page only, not for all pages.

Reply:
Done

Reply:
Seems like your fix is not enough.
![image](https://user-images.githubusercontent.com/7396837/186934904-c3de9dc8-b3af-4556-8401-3ab6073903d3.png)

My suggestion:
Take a look at `<main>` section. Apply the background for the element in case of Dashboard only.

Reply:
Done